cm_sdk/api/__init__.py

```diff
@@ -0,0 +1,4 @@
+__author__ = 'e0d'
+
+class APIException(Exception):
```

Comment:
This should probably have a more specific name to distinguish it from other api's exceptions. Also it should be defined elsewhere and exposed via `__all__`.